mon-pix/tests/integration/components/routes/register-form-test.js

```diff
@@ -312,5 +314,700 @@ describe('Integration | Component | routes/register-form', function() {
         expect(find('.register-form__error').innerHTML).to.equal(errorMessage);
       });
     });
+
+    describe('When student is already reconciled in the same organization', async function() {
+
+      describe('When student account is authenticated by email only', async function() {
+
+        it('should return a conflict error and display the error message related to the short code S61)', async function() {
```

Comment:
Plutôt que `should return a conflict error and...` je mettrais directement `should display....` car ce que tu veux tester c'est que le bon message d'erreur s'affiche lorsque l'API te retourne une 409 uniquement. Le retour de l'API tu le mock donc je ne le ferais pas figurer dans la description du `it()`. S'applique aux autres descriptions similaires dans le fichier.
```suggestion
        it('should display the error message related to the short code S61)', async function() {
```